packages/dashboard/src/App.tsx

```diff
@@ -26,6 +28,7 @@ function App(): JSX.Element {
                     <Route path="/" element={<Layout />}>
                       <Route index element={<Navigate to="/rpcs" replace />} />
                       <Route path="rpcs" element={<RPCs />} />
+                      <Route path="settings" element={<Analytics />} />
```

Comment:
It seems to me `/settings` should lead to `<Settings />`, not `<Analytics />`.

Reply:
Good point. Will make the change.